salt/docs/init.sls

```diff
@@ -9,7 +9,7 @@ include:
 {% set userdir = '/home/' + user %}
 {{ create_user(user, authorized_keys=pillar.ssh.docs) }}
 
-allow {{ userdir }} access:
+allow Apache access to {{ userdir }}:
   file.directory:
     - name: {{ userdir }}
```

Comment:
Ah sorry I missed this, I will test and secure this path further if we can.

Reply:
I have tested this and Apache needs access permissions on all of the parent directories.
So we cannot make the path in this state more specific.

Alternatively, we may be able to workaround this by changing the `www-data` group to allow access to the `ocds-docs` user.
This would be a bigger change that effects all servers so I think this is best in it's own issue.

Reply:
All good - I've added a comment to the Salt file.